### PR TITLE
Krhubert/cli

### PR DESCRIPTION
**Pull request by NicolasMahe**

### CODE REVIEW

File: cli/main.go [hunk 1]
```diff
@@ -5,12 +5,12 @@ import (
 	"github.com/mesg-foundation/core/cmd/utils"
 )
 
-// version of this release. Will be replaced automatically when compiling in CI
-var version = "vX.X.X"
+// Version of this release. Will be replaced automatically when compiling in CI
+var Version = "vX.X.X"
```
Comment (antho1404) [inline cli/main.go]: why make this variable accessible ?

Comment (ilgooz) [reply cli/main.go]: I think there is no rules about public variables when using ldflag. @krhubert might converted version to public maybe to document that Version will be accessed and set from outside, in this case by ldflag.

Comment (NicolasMahe) [reply cli/main.go]: Let's not forget to edit the build script:
https://github.com/mesg-foundation/core/blob/dev/scripts/build-cli.sh#L7

**Comment by NicolasMahe** [comment]
@mesg-foundation/core a modification should be test or done on: (see comment of this PR)
https://github.com/mesg-foundation/core/blob/dev/scripts/build-cli.sh#L7

**Comment by NicolasMahe** [comment]
Closing this PR in favor of https://github.com/mesg-foundation/core/pull/343/files

**Pull request closed without merging**